client/src/components/Draftail/CommentableEditor/CommentableEditor.js

```diff
@@ -20,21 +20,60 @@ function usePrevious(value) {
 class DraftailInlineAnnotation {
     constructor(field) {
       this.field = field;
-      this.inlineRefs = new Set()
+      this.inlineRefs = new Map();
+      this.focusedBlockKey = '';
+      this.cachedMedianRef = null;
     }
-    addRef(ref) {
-      this.inlineRefs.add(ref);
+    addRef(ref, blockKey) {
```

Comment:
The name ``ref`` isn't very clear since it could point to anything. Are they pointing to "decorators"? If so, could we rename this to ``decoratorRef``?

Reply:
Done!